### DISCUSSION TITLE
How to change sender's computer IP to email server IP

### DISCUSSION THREAD

**Discussion by moazam1** [postalserver/postal#1869] [Help with using Postal]
Hi Devs,

Thank you so much for amazing work. I just started using postal and I love the way how it works.

I am sending test emails from web ui and seeing following headers:

from web-ui (cpc-2-0-cust390.19-1.cable.virginm.net [xx.xx.xx.xxxx]) by Postal with HTTP; Tue, 15 Feb 2022 17:11:18 +0000

I was wondering how can I change IP sent in headers so instead of using my IP it uses email server IP?

Also, how can I restart postal instance automatically if instance is failing or server reboots?

Appreciate your help.

**Comment by willpower232** [comment]
I think you'd have to change Postals code to change that message. I don't think there is another way.

Re restarting, have a look at #1610 for some ideas
